web-ui/src/main/resources/catalog/components/viewer/wmsimport/partials/esrilayer.html

```diff
@@ -0,0 +1,25 @@
+<li data-ng-class='::!isParentNode ? "leaf" : ""'>
+  <div class="flex-col">
+    <div class="flex-row width-100 flex-align-stretch">
+      <div class="flex-col flex-grow">
+        <a href=""
+           class="truncate"
+           data-ng-click="addLayer(null, $event)"
```

Comment:
`addLayer()`  would be enough here I think

Reply:
ok, will update, copied from similar existing code for WMS services.